package.json

```diff
@@ -54,7 +54,8 @@
     "postcss-modules-scope": "^2.1.1",
     "postcss-modules-values": "^3.0.0",
     "postcss-value-parser": "^4.0.2",
-    "schema-utils": "^2.6.0"
+    "schema-utils": "^2.6.0",
+    "semver": "^7.1.2"
```

Comment:
https://github.com/npm/node-semver/blob/master/package.json#L36

I think we need to use lower version

Reply:
I can try and find one. Ya'll know that node 8 is End of Life right?

Reply:
We still support node@8 https://github.com/webpack-contrib/css-loader/blob/master/package.json#L16, In the near future there will be a release that will throw out support node@8, but right now it is breaking change, so we should avoid it

Reply:
yup, just wanted to make sure ya'll knew.